libraries/CMakeLists.txt

```diff
@@ -279,7 +287,11 @@ if (ENABLE_CAMP)
     JSON_FORTRAN_HOME=${CMAKE_BINARY_DIR}
     SUNDIALS_HOME=${CMAKE_BINARY_DIR}
   )
-  ExternalProject_Add(camp
+  # next couple lines added by D. Quevedo: Jan 29 2025
```

Comment:
This is thoughtful, but you don't need to add your name and date to your additions. One can find this info using `git blame`. :-)